File: C/unit_2/Fd.c
#include <stdio.h>
#include <math.h>
int main()
{
  float x, y, r, th;
  printf("Enter the Horizontal Cartesian Coordinates : \n");
  scanf("%f", &x);
  printf("Enter the Vertical Cartesian Coordinates : \n");
  scanf("%f", &y);
  r = sqrt(pow(x, 2) + pow(y, 2));
  th = atan(y / x);
  printf("The Polar Coordinates are given by : \n");
  printf("r = %d\n", r);
  printf("Angle = %d", (th * 180) / 3.14);
  return 0;
}

File: Java/unit_2/Fd.java
package Java.unit_2;

import java.util.Scanner;

public class Fd {
    public static void main(String[] args) {
        double x, y, r, th;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Horizontal Catresian Coordinates : \n");
        x = sc.nextDouble();
        System.out.println("Enter the Vertical Cartesian Coordinates :\n");
        y = sc.nextDouble();
        r = Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2));
        th = Math.atan(y / x);
        System.out.println("The Polar Coordinates Are Given by: \n r = " + r + " \n Angle = " + th * 180 / Math.PI);
        sc.close();
    }
}
